fda-model/paireddata/UncertainPairedData.cs

```diff
@@ -139,7 +139,7 @@ public IPairedData SamplePairedData(double probability)
                 }
                 if (pairedData.RuleMap[nameof(pairedData.Xvals)].ErrorLevel > ErrorLevel.Unassigned)
                 {
-                    Array.Sort(pairedData.Xvals);//bad news.
+                    throw new Exception("X values are not monotonically increasing");
```

Comment:
im not a fan of exceptions. can we use the message system?

another possiblity is "force monotonic in range" to make it increase.

Reply:
I will give it a try